### DISCUSSION TITLE
Ability to hide content between tags

### DISCUSSION THREAD

**Discussion by tororon1231** [LostRuins/koboldcpp#1137] [Ideas]
Add an option to hide content between tags (e.g. between thinking tags to make it more realistic).

**Comment by LostRuins** [comment]
This is possible with the regex replace feature, plus Display Only
Try this setup:

![image](https://github.com/user-attachments/assets/639f295d-282c-4061-86e4-7d9792624852)

**Reply by tororon1231** [reply]
Thanks. It worked!
